live-examples/js-examples/regexp/regexp-assertions.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var text = "A quick fox";
+var regexpAhead = /[a-z]+ (?=fox)/;
+console.log(text.match(regexpAhead));
+// expected output [ 'quick ']
```

Comment:
We have a `:` here, and the console output is a little different:

`// expected output: Array ["quick "]`

I know this sounds super-picky, but if we ever use "expected output" to test the examples (which I would love, one day) this kind of consistency will be important.

Reply:
I agree but also need to state that when testing the code in Fx & Chrome consoles, things are not always consistent. I'm not sure what would be the best baseline and what are the details of the "console" implementation of the interactive example "engine".